### DISCUSSION TITLE
Can @babel/types generate statements from source?

### DISCUSSION THREAD

**Discussion by shadowtime2000** [babel/babel#11996] [Q&A] [answered]
## Feature Request

- [ ] I would like to work on this feature!

**Describe the solution you'd like**
A function in `@babel/types` that allows you to add some code like `console.log` like this: `t.customFunction("console.log('It works!')");`

**Comment by JLHwung** [comment, marked as answer]
Babel supports AST templates in [`@babel/template`](https://babeljs.io/docs/en/babel-template#ast)
```js
import template from "@babel/template"
const ast = template.ast(`
  console.log('It works!')
`);
```
The `ast` is then a `CallExpression`.

**Reply by shadowtime2000** [reply]
I want this function to produce statements that could be used with a block statement

**Reply by shadowtime2000** [reply]
Never mind, `template.statement` works.
